cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaRebalanceStateMachineTest.java

```diff
@@ -78,6 +79,17 @@ public class KafkaRebalanceStateMachineTest {
                     .withBrokers(3)
                     .build();
 
+    private static final KafkaRebalanceSpec REMOVE_DISKS_KAFKA_REBALANCE_SPEC =
+            new KafkaRebalanceSpecBuilder()
+                    .withMode(KafkaRebalanceMode.REMOVE_DISKS)
+                    .withMoveReplicasOffVolumes(
+                            new BrokerAndVolumeIdsBuilder()
+                                    .withBrokerId(0)
+                                    .withVolumeIds(1)
+                                    .build())
+                    .withBrokers(3)
```

Comment:
this is a left over from some copy/paste, `brokers` field doesn't make sense when mode is remove disks.